cli/src/main/scala/org/scalajs/cli/Scalajsld.scala

```diff
@@ -68,6 +79,11 @@ object Scalajsld {
         .unbounded()
         .action { (x, c) => c.copy(cp = c.cp :+ x) }
         .text("Entries of Scala.js classpath to link")
+      opt[EntryPoint]('e', "execute")
+        .valueName("<full.name.Object.main>")
+        .unbounded()
+        .action { (x, c) => c.copy(entryPoints = c.entryPoints :+ x) }
+        .text("Execute the specified main method on startup")
```

Comment:
Ok, main method is fine with me. But can we make it `mainMethod` to be consistent with the other names?